tasks/parallel_verification.py

```diff
@@ -94,8 +94,12 @@ def run_impl(
         file_level_totals_match = True
         file_level_mismatched_files = []
 
-        # top level totals comparison
-        if parallel_report.totals.astuple() != serial_report.totals.astuple():
+        # top level totals comparison (ignoring session total, index 9)
+        parallel_tlt = list(parallel_report.totals.astuple())
+        parallel_tlt[9] = 0
+        serial_tlt = list(serial_report.totals.astuple())
+        serial_tlt[9] = 0
```

Comment:
Were these meant to be hard-coded? If they were, can you add a comment explaining why we want to hard-code this value?